canonical-web-and-design/maas-ui
# Migrate maas-ui to singlespa.

## Done
* Add root singlespa application.
* Migrate legacy routing to uirouter.
* Upgrade legacy vanilla to 2.10.1.
* Change legacy and ui builds to output UMD.
* Update cypress tests to reflect build changes.

## QA
* Ensure app runs in dev after a clean:
```
yarn clean-all
yarn install
cd proxy && yarn start
```
* Ensure app builds:
```
yarn clean-all
yarn build-all
```
